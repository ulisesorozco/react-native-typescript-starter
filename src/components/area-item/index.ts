import AreaItem from './area-item'
export default AreaItem
