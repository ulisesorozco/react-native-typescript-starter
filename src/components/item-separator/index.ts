import ItemSeparator from './item-separator'
export default ItemSeparator
