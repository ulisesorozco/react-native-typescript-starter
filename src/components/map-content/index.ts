import MapContent from './map-content'
export default MapContent
