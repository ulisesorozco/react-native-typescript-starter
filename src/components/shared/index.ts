export * from './text'
export * from './keyboard'
