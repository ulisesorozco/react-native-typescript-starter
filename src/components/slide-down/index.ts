import SlideDown from './slide-down'
export default SlideDown
