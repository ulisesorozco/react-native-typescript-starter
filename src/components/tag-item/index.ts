import TagItem from './tag-item'
export default TagItem
