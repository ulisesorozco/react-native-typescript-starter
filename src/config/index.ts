import { Text } from 'react-native'

if (__DEV__) {
  // If ReactNative's yellow box warnings are too much, it is possible to turn
  // it off, but the healthier approach is to fix the warnings.  =)
}
