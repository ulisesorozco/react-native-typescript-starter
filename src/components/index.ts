import MapContent from './map-content'
import SlideDown from './slide-down'
import InputBar from './input-bar'
import ItemSeparator from './item-separator'
import AreaItem from './area-item'
import TagItem from './tag-item'
import { Text, KeyboardSpacer } from './shared'

export { AreaItem, InputBar, ItemSeparator, MapContent, SlideDown, TagItem, Text, KeyboardSpacer }
