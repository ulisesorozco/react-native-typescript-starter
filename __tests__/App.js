import 'react-native'
import React from 'react'
import { Text } from '../src/components'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

it('Keyboard component renders correctly', () => {
  const tree = renderer.create(
    <Text text="test" />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
