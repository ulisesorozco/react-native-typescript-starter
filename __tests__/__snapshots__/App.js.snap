// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Keyboard component renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "rgb(255,255,255)",
      "fontFamily": "montserrat",
      "fontSize": 18,
      "textAlign": "center",
    }
  }
>
  test
</Text>
`;
